src/common/user-store.ts

```diff
@@ -80,6 +81,12 @@ export class UserStore extends BaseStore<UserStoreModel> {
     }
   }
 
+  async load(): Promise<void> {
+    await fileNameMigration();
```

Comment:
Why do we need to call this on every load? Migrations are usually executed during boot.

Reply:
... I don't remember why I did this. Good point.

Reply:
Could we add a comment here that explains why this needs to be called here?